push/templates/push/deletion.html

```diff
@@ -0,0 +1,25 @@
+{% extends "dashboard/base.html" %}
+{% load i18n %}
+
+{% block breadcrumbs %}
+<li><a href="{% url 'home' %}">{% trans "Home" %}</a></li>
+<li><a href="{% url 'push.landing' %}">{% trans "Push" %}</a></li>
+<li><a href="{% url 'push.list' %}">{% trans "Applications" %}</a></li>
+<li><a href="{% url 'push.details' app.id %}">{{ app.name }}</a></li>
+<li><span class="show-for-sr">{% trans "Current:" %} </span>{% trans "Delete" %}</li>
+{% endblock %}
+
+{% block content %}
+        <form id="push-app-deletion" action="{% url 'push.deletion' app.id %}" method="POST">
+            <fieldset>
+            {% csrf_token %}
+                <legend>{% blocktrans %}Are you sure you want to delete?{% endblocktrans %}</legend>
+                <dl>
+                    <dt>{% trans "App Name:" %}</dt><dd>{{ app.name }}</dd>
+                    <dt>{% trans "VAPID Key:" %}</dt><dd>{{ app.vapid_key }}</dd>
+                    <dt>{% trans "VAPID Key Status:" %}</dt><dd>{{ app.vapid_key_status }}</dd>
+                </dl>
```

Comment:
Nice! This makes more sense semantically, and it looks great too. And if we decide we do prefer the table-like presentation, we can even use CSS to reproduce that.